client/src/www/model/platform_error.ts

```diff
@@ -216,14 +193,31 @@ export class PlatformError extends CustomError {
     }
     return result;
   }
+}
 
-  /**
-   * Returns a JSON string of this error with all details and causes.
-   * @returns {string} The JSON string representing this error.
-   */
-  toJSON(): string {
```

Comment:
This is used when initiating a `PlatformError` from electron main process (not from Go) to renderer process: https://github.com/Jigsaw-Code/outline-apps/blob/master/client/electron/routing_service.ts#L141

Reply:
Yes, we should move the logic to Go in the future. All errors should be triggered from Go.